src/quo/components/avatars/wallet_user_avatar/style.cljs

```diff
@@ -2,25 +2,29 @@
   (:require [quo.foundations.colors :as colors]))
 
 (defn- circle-color
-  [customization-color]
-  (colors/custom-color customization-color 50 20))
+  [customization-color theme]
+  (if (= customization-color :neutral)
```

Comment:
nit:
can this be 
```
(let [neutral? (not customization-color)]
(if  neutral? ...)
```
or someway of not putting some value in `customization-color` which would not actually be resolved, it could be a bit confusing and slightly breaks the custom-color mechanism this way.

Reply:
or perhaprs we should add 50/60 values of neutral to colors map in colors cljs so they can be resolved like success, and danger. probably this makes sense more as it's quite common for like toasts or info messages etc

Reply:
I'll check, the thing is neutral color in this case makes use of neutral-80-opa-5 for light, and  white-opa-5 for dark, so it is a weird case.